pages/Sidebar.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+const Sidebar = () => {
+  const [selectedItem, setSelectedItem] = useState('About');
+
+  const handleItemClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  return (
+    <div className='fixed top-0 left-0 h-screen w-1/5 box-border bg-[#081f2e] '>
+      <div className='flex-row h-70 text-center my-4'>
+        <img src='/images/joemon1.jpg' alt="Image Description" className="w-[10rem] h-[10rem] mx-auto rounded-full " />
+        <h3>Joemon Mathew</h3>
+      </div>
+      <div></div>
+      <ul className='flex flex-col items-center h-full  mr-1 space-y-5 bg-[#020107] box-border '>
+        <li className='mt-5'>
+          <a
+            onClick={() => handleItemClick('About')}
+            className={`cursor-pointer px-20 text-lg ${selectedItem === 'About' ? 'text-white bg-[#081f2e]' : 'text-gray-300'}`}
+          >
+            About
+          </a>
```

Comment:
Use Next.js `Link` component instead of `a` tag

Also, change the URL based on the screen. eg: yourwebsite/about when the user clicks About.